tasks/quality_control/basic_statistics/task_nanoplot.wdl

```diff
@@ -46,8 +50,12 @@ task nanoplot {
     File nanoplot_html = "~{samplename}_NanoPlot-report.html"
     File nanoplot_tsv = "~{samplename}_NanoStats.txt"
     Int num_reads = read_int("NUMBER_OF_READS")
+    Float median_readlength = read_float("MEDIAN_READ_LENGTH")
     Float mean_readlength = read_float("MEAN_READ_LENGTH")
+    Float stdev_readlength = read_float("READ_LENGTH_STDEV")
+    Float n50 = read_float("N50")
     Float mean_q = read_float("MEAN_QUAL")
+    Float median_q = read_float("MEDIAN_QUAL")
```

Comment:
is median quality a useful metric? I think mean probably does a decent job at this since there's not going to be crazy outliers

Reply:
same question for standard deviation in read length

Reply:
also maybe i'm missing the plot but isn't n50 most useful in the contig context rather than read?

Reply:
Thank you so much for the review @sage-wright ! Read-level n50 was actually the one metric that was asked for in #355! I went ahead and exposed all the metrics computed by nanoplot directly to the data table, but @michellescribner can shed some light on whether this was the most appropriate path to follow. :) 
I thought that more was better in this case!

Reply:
Read level n50 was a specific request from a laboratory, so I'm all for it. Gratuitous QC metrics is my preference based on what we have heard from labs